findspam.py

```diff
@@ -2413,6 +2414,75 @@ def religion_troll(s, site):
             r'\bICQ[ :]{0,5}\d{9}\b|\bwh?atsa+pp?[ :+]{0,5}\d{10}',
             stripcodeblocks=True)
 
+
+# Homoglyph obfuscation, used both by trolls and spammers
+@create_rule("obfuscated word", max_rep=50, stripcodeblocks=True)
+def obfuscated_word(s, site):
+    obfuscation_keywords = [
+        # Trolls && cussing
+        "c" + "ock",
+        "c" + "unt",
+        "d" + "ick",
+        "f" + "uck",
+        "mother" + "f" + "ucker",
+        "p" + "enis",
+        "p" + "ussy",
+        "w" + "hore",
```

Comment:
Why are you breaking these words up? Functionally, it doesn't affect it, but it's unclear why the actual words shouldn't be here. Yes, they are offensive words, but that's what we're working with.